### ISSUE TITLE
How to use DSSIM loss function in model.

### ISSUE THREAD

**Issue by AYeshwanth** [the-moliver/kfs#19]
Can you explain how to use DSSIM as a loss function while training the model in keras ( or tensorflow ). I am trying to use SSIM to help in denoising OCT images.
